### ISSUE TITLE
[Bug]: Config file not loaded for models that use v-prediction mode

### ISSUE THREAD

**Issue by AndrewLieber** [lllyasviel/stable-diffusion-webui-forge#104]
### Checklist

- [X] The issue exists after disabling all extensions
- [X] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [X] The issue exists in the current version of the webui
- [X] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

Zero terminal SNR models like EasyFluff (https://huggingface.co/zatochu/EasyFluff) require a config file to use in v-prediction mode, with Forge the config file does not appear to load.
Left is using Auto1111 where the config is loaded along with the model, right is Forge
![92853-3470808785-not furry, (by mamaloni, by koahri_0 7), (by funitarefu_0 4), 1girl, brown hair, bob cut, outdoors, windy, floral print, puffy s](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/121382703/d663722e-a2f1-4167-a114-3e2a6e9f6de7)

### Steps to reproduce the problem

Place model and config file in models folder
Generate image

### What should have happened?

Config file should be loaded

### Sysinfo

[sysinfo-2024-02-07-12-27.json](https://github.com/lllyasviel/stable-diffusion-webui-forge/files/14193997/sysinfo-2024-02-07-12-27.json)

### Console logs

```Shell
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: f0.0.10-latest-64-g257ac265
Commit hash: 257ac2653a565672b280f2851f37b1ba6e546548
Launching Web UI with arguments:
Total VRAM 8192 MB, total RAM 8156 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce GTX 1070 : native
VAE dtype: torch.float32
Using pytorch cross attention
ControlNet preprocessor location: F:\Forge\webui\models\ControlNetPreprocessor
Loading weights [821628644e] from F:\Forge\webui\models\Stable-diffusion\EasyFluffV11.2.safetensors
2024-02-07 04:11:57,149 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
model_type EPS
UNet ADM Dimension 0
Startup time: 27.3s (prepare environment: 8.0s, import torch: 8.1s, import gradio: 2.3s, setup paths: 2.0s, initialize shared: 0.2s, other imports: 1.5s, load scripts: 3.1s, create ui: 1.3s, gradio launch: 0.6s).
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_l.logit_scale'}
loaded straight to GPU
To load target model BaseModel
Begin to load 1 model
Loading VAE weights specified in settings: F:\Forge\webui\models\VAE\furception_vae_1-0.safetensors
To load target model SD1ClipModel
Begin to load 1 model
Moving model(s) has taken 0.13 seconds
Model loaded in 23.5s (load weights from disk: 1.0s, forge solving config: 0.2s, forge instantiate config: 0.2s, forge load real models: 18.6s, load VAE: 2.5s, calculate empty prompt: 0.9s).
                                  To load target model BaseModel
Begin to load 1 model00:00, ?it/s]
unload clone 1
Moving model(s) has taken 10.51 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 28/28 [00:31<00:00,  1.14s/it]
To load target model AutoencoderKL█████████████████████████████▉                       | 28/43 [00:41<00:15,  1.04s/it]
Begin to load 1 model
Moving model(s) has taken 4.37 seconds
Upscale script freed memory successfully.
tiled upscale: 100%|███████████████████████████████████████████████████████████████████| 20/20 [00:06<00:00,  3.29it/s]
To load target model BaseModel
Begin to load 1 model
Moving model(s) has taken 4.34 seconds
100%|██████████████████████████████████████████████████████████████████████████████████| 15/15 [01:35<00:00,  6.36s/it]
To load target model AutoencoderKL█████████████████████████████████████████████████████| 43/43 [02:35<00:00,  6.37s/it]
Begin to load 1 model
Moving model(s) has taken 0.84 seconds
Total progress: 100%|██████████████████████████████████████████████████████████████████| 43/43 [02:42<00:00,  3.79s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 43/43 [02:42<00:00,  6.37s/it]
```

**Comment by BurnZeZ** [comment]
@lllyasviel Was this ever resolved?
I’m I have yet to get forge to use the Zero Terminal SNR noise scheduler either, whether through settings or override.
It’d be nice if at least one person could confirm they can generate images with both schedulers with [current tip](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/29be1da7cf2b5dccfc70fbdd33eb35c56a31ffb7).
